packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -447,6 +449,7 @@ class _InspectorOverlayLayer extends Layer {
       overlayRect: overlayRect,
       selected: new _TransformedRect(selected),
       tooltip: selected.toString(),
+      textDirection: TextDirection.ltr,
```

Comment:
Maybe add a TODO about localization?

Reply:
I don't think we'll ever localize the debug strings.